sdk/documentintelligence/azure-ai-documentintelligence/samples/sample_analyze_layout_and_sort_paragraphs.py

```diff
@@ -0,0 +1,74 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+
+"""
+FILE: sample_analyze_custom_documents_and_read_table_cells.py
+
+DESCRIPTION:
+    This sample demonstrates how to analyze a document with a prebuilt-read built model
+    and get the sorted paragraphs.
+
+USAGE:
+    sample_analyze_layout_and_sort_paragraphs.py
+
+    Set the environment variables with your own values before running the sample:
+    1) DOCUMENTINTELLIGENCE_ENDPOINT - the endpoint to your Document Intelligence resource.
+    2) DOCUMENTINTELLIGENCE_API_KEY - your Document Intelligence API key.
+"""
+
+
+def analyze_document_and_sort_paragraphs():
+    import os
+    from azure.core.credentials import AzureKeyCredential
+    from azure.ai.formrecognizer import DocumentAnalysisClient
+
+    endpoint = os.environ["DOCUMENTINTELLIGENCE_ENDPOINT"]
+    key = os.environ["DOCUMENTINTELLIGENCE_API_KEY"]
+    # sample document
+    formUrl = "https://raw.githubusercontent.com/Azure-Samples/cognitive-services-REST-api-samples/master/curl/form-recognizer/sample-layout.pdf"
+
+    document_analysis_client = DocumentAnalysisClient(
+        endpoint=endpoint, credential=AzureKeyCredential(key)
+    )
+
+    poller = document_analysis_client.begin_analyze_document_from_url(
+        "prebuilt-read", formUrl
+    )
+
+    result = poller.result()
+    result.paragraphs.sort(key=lambda p: p.spans[0].offset)
+    print("----Verify the order of paragrahps------")
+    for idx, paragraph in enumerate(result.paragraphs):
+        print(f"...paragraph:{idx}, offset:{paragraph.spans[0].offset}")
+
+    print("----------------------------------------")
```

Comment:
This only adds a very simple code to enumerate the paragraphs, should we simply extend in `sample_analyze_layout.py`?

Reply:
moved, thx